regression_tests/cases/distillation_settings/distillation_times.1.case.sh

```diff
@@ -4,7 +4,11 @@ include "qelib1.inc";
 
 qreg q[2];
 
-t q[1];
```

Comment:
The problem here was in the waiting mechanism. Incidentally this CR fixes it, but modifying this test not to cover the waiting feature